features/migrations/0022_auto_20200415_1432.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.0.3 on 2020-04-15 11:32
```

Comment:
please, avoid autogenerated timestamp names for you migrartion files. verbose names in their case help navigate the codebase much better